Lib/test/test_email/test_email.py

```diff
@@ -3204,6 +3204,12 @@ def test_getaddresses_embedded_comment(self):
         addrs = utils.getaddresses(['User ((nested comment)) <foo@bar.com>'])
         eq(addrs[0][1], 'foo@bar.com')
 
+    def test_getaddresses_header_obj(self):
+        """Test the handling of a Header object."""
+        eq = self.assertEqual
```

Comment:
I don't think we should create an alias if we are going to use this only once :)

Reply:
Agreed.  I use this alias all the time, but it needs at least 2 and usually 3 uses to pay for itself.